presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeGeneralQueries.java

```diff
@@ -69,4 +69,7 @@ public void testTopN() {}
     @Override
     @Ignore
     public void testInsertIntoSpecialPartitionName(){}
+    @Override
+    @Ignore
+    public void testAnalyzeStats() {}
```

Comment:
Why is this ignored? Would you add a comment and perhaps refer to a GitHub issue that explains the problem.

Reply:
@aditi-pandit @karteekmurthys Would take a look at this question? I'm still wondering about this.

Reply:
I have opened an issue that i faced during testing. Please refer L73.